.github/workflows/release_prep.yml

```diff
@@ -0,0 +1,15 @@
+name: "Release Prep"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Module version to be released. Must be a valid semver string. (1.2.3)"
+        required: true
+
+jobs:
+  release_prep:
+    uses: "puppetlabs/cat-github-actions/.github/workflows/module_release_prep.yml@main"
```

Comment:
not actually sure pdk-templates can fit the reusable templates because of this step here https://github.com/puppetlabs/cat-github-actions/blob/6c374668d26258ac920cd6e11f481600b9bd2e76/.github/workflows/module_release_prep.yml#L35. Gem release prep has a similar step

Reply:
if it doesn't fail on these steps we could use it i suppose

Reply:
Ok yeh........
Think I'll need to just copy the reusable over and alter it.